cms/templates/admin/cms/page/tree/base.html

```diff
@@ -107,8 +107,8 @@ <h1>
                                     <span class="label">{% trans "Sites" %}</span>
                                 </li>
                                 {% for site in tree.sites %}
-                                    <li{% if site.pk == tree.site.pk %} class="active"{% endif %}>
-                                        <a href="#{{ site.pk }}" class="js-cms-pagetree-site-trigger" data-id="{{ site.pk }}">{{ site.name }}</a>
+                                    <li{% if site.pk|unlocalize == tree.site.pk|unlocalize %} class="active"{% endif %}>
```

Comment:
The if statement works without the `unlocalize`. Can you remove it here?

Reply:
... and done.